core/src/main/scala/caliban/validation/ValueValidator.scala

```diff
@@ -111,14 +115,14 @@ object ValueValidator {
         }
     }
 
-  def validateEnum(value: String, inputType: __Type, errorContext: String): IO[ValidationError, Unit] = {
+  def validateEnum(value: String, inputType: __Type, errorContext: String): EReader[Any, ValidationError, Unit] = {
```

Comment:
How about making a type alias for that so we only specify the `A`?

Reply:
So it should have `Context` fixed as the type?